pkg/controller/controller.go

```diff
@@ -774,6 +774,10 @@ func (s *Service) ControllerGetCapabilities(ctx context.Context, request *csi.Co
 		csi.ControllerServiceCapability_RPC_LIST_SNAPSHOTS,
 		csi.ControllerServiceCapability_RPC_CLONE_VOLUME,
 		csi.ControllerServiceCapability_RPC_EXPAND_VOLUME,
+		csi.ControllerServiceCapability_RPC_GET_VOLUME,
+		csi.ControllerServiceCapability_RPC_LIST_VOLUMES_PUBLISHED_NODES,
```

Comment:
please remove getvolume, listvolumepublishednodes, volumecondition from this block